storage/Devices/PartitionImpl.cc

```diff
@@ -269,6 +307,34 @@ namespace storage
 
 
     void
+    Partition::Impl::set_boot(bool boot)
+    {
+	if (!is_msdos(get_partition_table()))
+	    ST_THROW(Exception("set_boot only supported for Msdos"));
+
+	if (boot && !Impl::boot)
+	{
+	    PartitionTable* partition_table = get_partition_table();
+
+	    for (Partition* partition : partition_table->get_partitions())
```

Comment:
But it can be done with cfdisk. So it's possible. For example, in [wikipedia's article](https://en.wikipedia.org/wiki/Boot_flag) it says "some modern BIOS test if the bootflag of **at least one** partition is set". Which also seems to imply that is possible.

The fact that we use parted under the hood and that parted cannot do it is just an implementation detail. As long as there are common tools out there allowing it, we should document that our library doesn't.